app/views/clearance/sessions/_form.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_for :session, url: session_path do |form| %>
+  <div class="input-group">
+  <span class="input-group-addon" id="basic-addon1">E-mail</span>
+    <%= form.text_field :email %>
+  </div>
+
+  <div class="input-group">
+  <span class="input-group-addon" id="basic-addon1">Password</span>
+    <%= form.text_field :password %>
+  </div>
+
+  <div class="submit-field">
+    <%= form.submit %>
+  </div>
+
+<% end %>
```

Comment:
Oooh. If this is not needed, I would be okay with removing it in this PR as a separate commit, actually. It's borderline on what is "housekeeping" and not. 😀